src/assets/css/style.css

```diff
@@ -32,18 +36,7 @@
 }
 
 body {
-  width: 100vw;
-  height: 100vh;
-  margin: 0;
-  overflow: hidden;
-  grid-template-columns: auto 1fr auto;
-  grid-template-rows: auto 1fr auto;
-  background-color: var(--bg-color);
```

Comment:
Can you justify these CSS changes? The first 4 rows are mapped to tailwindcss classes, but other ones are either modified (background and text color changes to not follow css var) or deleted.